bootstrap/storageclass/helmfile/values/nfs-client-provisioner.yaml.gotmpl

```diff
@@ -1,12 +1,14 @@
-resources: {{- toYaml .Values.nfsProvisioner.resources | nindent 2  }}
+resources:    {{- toYaml .Values.nfsProvisioner.resources | nindent 2  }}
 nodeSelector: {{- toYaml .Values.nfsProvisioner.nodeSelector | nindent 2 }}
-affinity: {{- toYaml .Values.nfsProvisioner.affinity | nindent 2 }}
-tolerations: {{- toYaml .Values.nfsProvisioner.tolerations | nindent 2 }}
+affinity:     {{- toYaml .Values.nfsProvisioner.affinity | nindent 2 }}
+tolerations:  {{- toYaml .Values.nfsProvisioner.tolerations | nindent 2 }}
 
 storageClass:
-  defaultClass: true
+  defaultClass: {{ .Values.storageClasses.nfs.isDefaultClass }}
```

Comment:
Perhaps `storageClasses.default` isn't a very good name since this storageclass is used for persistent volumes in our helm charts, and so it's does not have to be a "default" storageclass in kubernetes.
You could imagine a scenario where you'd want to use something like `storageClasses.default: gcp-sd` but still set the nfs storageclass as default in kubernetes.

Reply:
Maybe, if you want to have two variables, one for our helm charts and one to set default in kubernetes, then that's ok with me.

Reply:
Yeah, I'm not sure. Maybe we could leave it the way it is right now and schedule a larger discussion on how we want to handle storageClasses going forward?

Reply:
Sounds good. I still think you can remove `storageClasses.nfs.isDefaultClass`

Reply:
Okay, I don't agree, but sure.

Reply:
Shouldn't the one set as default just be default in kubernetes and then if a chart wants to use a specific storage class you set it to something else than the default?

E.g.
The kubernetes default is `nfs-client` but we want to use `gcp-sd` for all our applications. Then we should set all our applications to choose a specific one (not use the default storageclass).

Reply:
I mean Yes.
As we discovered and discussed last week, your not forced to just one default storageclass in kubernetes, so there is nothing inherently with having storageClasses.*.isDefafultClass.
And also this `storageClasses.default` was just a simple solution for us to select a storageclass to use use in some of our helm charts, like kube-prometheus-stack.